api/version.go

```diff
@@ -14,6 +14,20 @@ import (
 
 // Version represents the API handler to
 // report the version number for Vela.
+// swagger:operation GET /version router Version
```

Comment:
Thoughts on moving the function description comments below the swagger specific comments?

My thought process is that it might help (or be nice) to remain consistent 👍

Reply:
To clarify - do something similar to login.go?
https://github.com/go-vela/server/blob/4a8063d440152c6a580ff9466a68bd400489bc9d/api/login.go#L77-L78

Reply:
@meinenec Yes that is correct 👍